Source/Core/Plugins/EventProcessor/Default/20_ErrorPlugin.cs

```diff
@@ -28,6 +29,7 @@ public class ErrorPlugin : EventProcessorPluginBase {
             if (context.HasProperty("UserNamespaces"))
                 userNamespaces = context.GetProperty<string>("UserNamespaces").SplitAndTrim(',');
 
+            AddManualStackingKeyToSignatureIfExists(context);
```

Comment:
I was thinking that this would be a separate step in the pipeline because it really has nothing to do with errors. It can be applied to any type of event.